src/requirements.txt

```diff
@@ -1,3 +1,6 @@
 apache-airflow[crypto,postgres]==1.10.5
--e git+https://github.com/Amsterdam/GOB-Core.git@v0.8.10#egg=gobcore
 pika==0.12.0
+colour==0.1.5
+flake8==3.5.0
```

Comment:
in the future we might want to split test and production requirements.

Reply:
Leave it like this for now, it is like any other GOB module